client/js/socket-events/init.js

```diff
@@ -12,6 +12,10 @@ const utils = require("../utils");
 socket.on("init", function(data) {
 	$("#loading-page-message, #connection-error").text("Rendering…");
 
+	// Store information about logged user in state, such as its role (admin or
+	// regular user)
+	$("body").data("current-user", data.currentUser);
```

Comment:
`document.body` damn it.

Reply:
> document.body damn it.

Goddamn, why do I always do that?! Fixing.

Reply:
> Should we wrap the data.currentUser in our local-user object you've just created for the server.

I didn't want to commit too much into this `LocalUser` solution just yet, in case we actually don't want it. I just timidly brought the idea, and if we do that, I'm happy to use it more.

That said, this is tricky: it is actually sent as a `LocalUser` from the server. I'm guessing when it goes through the socket, it's converted to JSON or something. I'm not against re-wrapping it in a `LocalUser` once it hits the client, but that requires some sharing of models code between server and client.
At the moment, we just don't do that (I know you mentioned that in an earlier discussion). I agree, but if we want to do it cleanly (i.e. having a shared location, library-like or something), that requires some refactoring, or we'll end up having some `require("../src/[...]")` on the client, which would just add to the `require("../client/[...]")` 💩  we have on the server 😁 

So in short, totally in favor, but we need to be smart. Maybe for a future PR? I don't know, no strong opinion, I just don't want this PR to solve many issues, be hard to review, rebase, etc.

Reply:
> I didn't want to commit too much into this LocalUser solution just yet, in case we actually don't want it. 

Yeah, that's fair. But I think we are probably going to go with it.

> or we'll end up having some require("../src/[...]") on the client

I agree it's less than ideal, but surely there's nothing wrong with just going for this in this PR, it'll be really easy to do for you just now, and then we can have another PR to refactor the common code into a different directory or another way of doing it.  I just see that we are doing the same thing in the client as we do in the localuser (to tell if the user is admin or not) and it would be nice if we were to start off without that.